tgui/packages/tgui/constants.ts

```diff
@@ -143,7 +143,7 @@ const GASES = [
     path: '/datum/gas/nitrogen',
     name: 'Nitrogen',
     label: 'N₂',
-    color: 'red',
+    color: 'yellow',
```

Comment:
careful, this does not seem to be react related

Reply:
ya i cherrypicked this manually